examples/route-mirror/verify.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+export NAME=route-mirroring
+export PORT_PROXY="${FRONT_PROXY_PORT_PROXY:-11820}"
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+
+run_log "Static mirror cluster: localhost:${PORT_PROXY}/service/1"
+wait_for 20 bash -c "responds_with \"Hello from behind Envoy (service 1)!\" http://localhost:${PORT_PROXY}/service/1"
```

Comment:
remove `wait_for`

Reply:
I seem to need it - the test fails without it. I also see why i think - we need some time for the envoy service to be up and running, I changed it to 10s and that seems to be good enough.

Reply:
its not so much about the length of time - if you need it, it suggests something is not coming up correctly - Envoy should be up by the time docker-compose returns

Reply:
What do you think of making use DELAY as documented [here](https://github.com/envoyproxy/envoy/blob/main/examples/DEVELOPER.md#slow-starting-docker-compositions)?

Reply:
> Envoy should be up by the time docker-compose returns
It likely is, but there is a race here - as we use `docker-compose up -d`.

Reply:
same thing - `DELAY` was added originally and i have been removing it in favour of compositions that come up correctly

im struggling to understand why its failing for you it seems to work for me ok - ill review again later and test then

Reply:
> but there is a race here

what race ?

Reply:
> im struggling to understand why its failing for you it seems to work for me ok - ill review again later and test then

That is strange. i have had success rate of 1/20 times so far without the `wait_for`.

Reply:
FWIW, `sleep 1` is sufficient for the tests to pass locally..

Reply:
I have pushed https://github.com/envoyproxy/envoy/pull/23336/commits/014290ffe49ddc26761e3698834af3496b923697 to see what happens in CI without `wait_for`.